optics-core/src/Optics/Setter.hs

```diff
@@ -128,10 +142,11 @@ set o = over o . const
 
 -- | Apply a setter, strictly.
 --
--- TODO DOC: what exactly is the strictness property?
---
+-- The new value will be forced if and only if the optic traverses at
+-- least one target. If forcing the new value is inexpensive, then it
+-- is cheaper to do so manually and use 'set'.
```

Comment:
I wonder if we could easily test this, since we didn't before and apparently didn't get it right...

Reply:
I'm not sure what the intention is here. I'm just documenting what it actually does.

Reply:
Or were you referring to testing strictness generally? `containers` does some of that; I haven't checked if there's traversal strictness testing, but if so that could be copied maybe.